src/sections/ProjectSection.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import gql from 'graphql-tag';
 import EditableText from '../forms/EditableText';
 import EditableMarkdown from '../forms/EditableMarkdown';
+import EditableSkills from '../forms/EditableSkills';
 import ProjectIssues from './ProjectIssues';
 
 const ProjectSection = ({ project, editing, onFormDataChange }) => {
```

Comment:
This whole edit project section needs some styling, but the functionality is still there and it doesn't undermine this PR I don't think